qiskit/providers/aer/library/set_instructions/set_matrix_product_state.py

```diff
@@ -0,0 +1,76 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Instruction to set the state simulator state to a matrix.
+"""
+
+from qiskit.circuit import QuantumCircuit, Instruction
+from qiskit.extensions.exceptions import ExtensionError
+from ..default_qubits import default_qubits
+
+
+class SetMatrixProductState(Instruction):
+    """Set the matrix product state of the simulator"""
+
+    _directive = True
+
+    def __init__(self, mps_state):
+        """Create new instruction to set the matrix product state of the simulator.
+
+        Args:
+            mps_state (mps_container_t): a matrix product state.
```

Comment:
This is referring to a C++ type. It should probably just be `matrix_product_state`, and in the docstring describe the format or reference the save instruction documentation if it is described there

Reply:
Done

Reply:
Done.